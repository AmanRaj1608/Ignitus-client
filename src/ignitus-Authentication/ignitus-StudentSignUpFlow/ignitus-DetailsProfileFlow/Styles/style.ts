import styled from '@emotion/styled';
import { Container } from '../../../../ignitus-Shared/ignitus-DesignSystem/shared';

import {
  Heading5,
  Paragraph,
  Heading6,
  minimumWidthQuery,
  DefaultIconInput,
  DefaultCheckbox,
} from '../../../../ignitus-Shared';
import { TitleProps, IconProps } from '../types';
import { Icon as I } from '../../../../ignitus-Shared/ignitus-Utilities/Components/icon';
import * as C from '../../../../ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/colors';
import * as F from '../../../../ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/fonts';

export const NewsletterImage = styled.img``;

export const TopMiddleSection = styled.div`
  margin-top: 1rem;
`;

export const ParentContainer = styled(Container)`
  margin: 10rem auto;
  background: ${C.White};
  box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, 0.1);
  border-radius: 2rem;
`;

export const TopSection = styled.div`
  display: flex;
  justify-content: flex-end;
`;
export const MiddleSection = styled.div`
  padding: 2rem;
`;
export const ElementContainer = styled(Container)`
  margin: 1.5rem 0;
`;

export const Title = styled(Heading5)<TitleProps>`
  padding: 0.5rem 2rem;
  font-weight: ${props =>
    props.fontStyle === 'paragraph' ? F.Medium : F.Bold};
  font-size: ${props => (props.fontStyle === 'paragraph' ? F.SM : F.MD)};
  color: ${props =>
    props.fontStyle === 'paragraph' ? C.GreyText : C.IgnitusBlue};
`;

export const TitleWrapper = styled.div`
  display: flex;
  flex-direction: row;
`;

export const Icon = styled(I)<IconProps>`
  width: 1.5rem;
  fill: ${props => props.color};
  margin: 0 0.5rem;
  background-color: ${props => props.background};
  border-radius: 0.3rem;
`;

export const Content = styled(Container)`
  margin-left: 3.6rem;
  margin-top: 1rem;
`;

export const University = styled(Heading5)`
  font-weight: ${F.SemiBold};
  margin-right: 2rem;
`;

export const Address = styled(Paragraph)`
  font-size: ${F.MD};
  color: ${C.GreySecondaryText};
  margin: 0.4rem 0;
`;

export const Date = styled(Paragraph)`
  margin: 0.4rem 0;
`;

export const ContributionsWrapper = styled.div`
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  flex-wrap: wrap;

  ${minimumWidthQuery[0]} {
    flex-direction: column;
  }
  ${minimumWidthQuery[2]} {
    flex-direction: column;
  }
  ${minimumWidthQuery[3]} {
    flex-direction: row;
  }
  ${minimumWidthQuery[4]} {
    flex-direction: row;
  }
  ${minimumWidthQuery[5]} {
    flex-direction: row;
  }
`;

export const RecommendationsRow = styled.div`
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
`;

export const LeftRow = styled.div`
  display: flex;
  flex-direction: row;
  justify-content: flex-start;
`;

export const RightRow = styled.div`
  display: flex;
  flex-direction: row;
  justify-content: flex-end;
`;
export const Avatar = styled.img`
  margin: 1rem;
  width: 9rem;
  border-radius: 50%;
`;
export const Name = styled(Heading5)``;
export const Designation = styled(Heading6)`
  margin: 0.4rem 0;
`;

export const BottomSection = styled.div`
  text-align: center;
  margin: 1rem;
  padding: 2rem;
`;

export const StyledTextArea = styled.textarea`
  background-color: ${C.GreyLight};
  padding-left: 1rem;
  color: ${C.GreySecondaryText};
  font-weight: ${F.SemiBold};
  font-size: ${F.MD};
  border: none;
  border-radius: 17px;

  ::placeholder {
    color: ${C.GreySecondaryText};
    font-weight: ${F.SemiBold};
    font-size: ${F.MD};
    opacity: 0.5;
`;

export const StyledArea = styled.div`
  background-color: ${C.GreyLight};
  padding-left: 1rem;
  border: none;
  border-radius: 17px;
  font-weight: ${F.SemiBold};
  font-size: ${F.MD};
  height: 10rem;
  width: 100%;
`;

export const StyledDefaultIconInput = styled(DefaultIconInput)`
  flex-direction: row-reverse;
  svg {
    margin-right: 0.4rem;
  }
`;

export const InputContainer = styled.div`
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  justify-content: space-between;
  padding: 1rem;
`;

export const StyledDefaultCheckbox = styled(DefaultCheckbox)`
  padding: 1rem 0;
`;
