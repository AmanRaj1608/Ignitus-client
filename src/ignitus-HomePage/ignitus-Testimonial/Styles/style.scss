@import './src/ignitus-Shared/styles/ignitus-tokens';
@import './src/ignitus-Shared/styles/ignitus-mixins';
$ignite-font-family-default: 'Raleway', sans-serif;
$ignite-color-blue-24: #192b65;
$ignite-font-weight-semibold: 600;
$ignite-font-size-sm: 16px;

.testimonial {
  .testimonials-title {
    font-family: $ignite-font-family-default;
    text-align: center;
    font-weight: 600;
    font-size: 22px;
    color: #000066;
    color: #000066;
  }

  .testimonial-wrapper {
    align-items: center;
    display: flex;
    margin: 0;
    flex-direction: column;
    justify-content: center;
  }

  .carousel-content {
    padding: 1em;
    text-align: center;
    width: auto;
  }

  .text-box  p {
    display: inline-block;
    font-size: 18px;
    font-style: italic;
    font-weight: 500;
    margin-left: 0.5em;
    margin-right: 0.5em;
    text-align: center;
  }


  .text-box::before {
    color: $ignite-color-blue-24;
    content: "\f10d";
    display: inline-block;
    font-family: "FontAwesome";
    vertical-align: top;
  }

  .text-box::after {
    color: $ignite-color-blue-24;
    content: "\f10e";
    display: inline-block;
    font-family: "FontAwesome";
    vertical-align: bottom;
  }

  .avatar {
    margin-bottom: 0.5em;

    img {
      border-radius: 100%;
      height: 15.5vh;
    }
  }

  .name {
    font-size: 18px;
    font-weight: $ignite-font-weight-semibold;
    margin-bottom: 0;
    text-align: center;
  }

  .designation {
    font-size: $ignite-font-size-sm;
    text-align: center;
  }

  .nav-avatar {
    align-items: center;
    display: flex;
    justify-content: center;
    min-height: 5.5vh;

    img {
      border-radius: 100%;
      height: 3.5vh;
      margin-right: 0.3em;
    }
    
    img:hover {
      border: 1px solid #192b65;
      height: 5vh;
      transition: ease 0.5s;
    }
  }

  @media only screen and (max-width: 596px) {
    display: none;
  }
}
