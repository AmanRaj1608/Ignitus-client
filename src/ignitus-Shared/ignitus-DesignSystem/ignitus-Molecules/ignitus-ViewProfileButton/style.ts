import styled from '@emotion/styled';
import * as C from '../../ignitus-Atoms/colors';
import * as B from '../../ignitus-Atoms/fonts';

export const Button = styled.div`
  align-items: center;
  background-color: ${C.IgnitusBlue};
  border-radius: 0.5rem;
  cursor: pointer;
  display:flex;
  height: 2rem;
  justify-content: center;
  width: 8rem;

  &:hover {
    border-radius: 0.5rem;
    box-shadow: 0 0 10px #000066;
    opacity: 1;
  }
`; 

export const Text = styled.div`
  color: ${C.White};
  font-size: ${B.XS};
`;