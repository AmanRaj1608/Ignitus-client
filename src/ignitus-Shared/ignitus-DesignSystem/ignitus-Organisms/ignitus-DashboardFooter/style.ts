import styled from '@emotion/styled';
import * as C from '../../ignitus-Atoms/colors';
import {flexibleColDiv, flexibleRowDiv} from '../../shared';
import {PrimaryFamily, XS} from '../../ignitus-Atoms/fonts';

export const Footer = styled.footer`
  background-color: ${C.White};
  box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.1);
  border-radius: 16px;
  padding: 1rem;
  display: inline-flex;
  flex-direction: column;
`;

export const Column = styled(flexibleColDiv)``;

export const FooterSections = styled(flexibleRowDiv)``;

export const FooterRow = styled.ul`
  color: ${C.GreySecondaryText};
  font-size: ${XS};
  font-family: ${PrimaryFamily};
  padding: 0.4rem;
  display: flex;
  flex-direction: row;
  li {
    padding: 0rem 1rem;

    a {
      &:link,
      &:visited {
        color: #3d3d3d;
      }
    }

    a {
      &:hover,
      &:active {
        color: ${C.IgnitusBlue};
      }
    }
  }
`;

export const Copyright = styled.p`
  color: ${C.IgnitusBlue};
  font-family: ${PrimaryFamily};
  font-size: ${XS};
  text-align: center;
  margin-top: 15px;
  margin-bottom: 0;
`;
