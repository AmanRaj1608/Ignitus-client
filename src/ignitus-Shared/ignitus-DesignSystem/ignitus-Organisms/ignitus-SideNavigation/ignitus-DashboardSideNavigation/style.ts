import styled from '@emotion/styled';
import {White, IgnitusBlue, GreyLight, boxShadowColor} from '../../../ignitus-Atoms/colors';
import {MD, Normal} from '../../../ignitus-Atoms/fonts';
import { flexibleColDiv } from '../../../shared';
import Icon from '../../../../ignitus-Utilities/Components/icon';


type Props = {
  border: boolean;
  marginTop: boolean;
  marginBottom: boolean;
};

export const SettingsContainer = styled(flexibleColDiv)`
  background: ${White};
  border-radius: 16px;
  box-shadow: 0 2px 4px 0 ${boxShadowColor};
  width: 12rem;
  height: 24rem;
`;

export const Layer = styled.div<Props>`
  border-bottom: ${props => (props.border ? `1px solid ${GreyLight}` : 'null')};
  color: ${IgnitusBlue};
  cursor: pointer;
  font-size: ${MD};
  font-weight: ${Normal};
  padding: 1rem;
  margin-top: ${props => (props.marginTop ? '0.5rem' : 'none')};
  margin-bottom: ${props => (props.marginBottom ? '0.5rem' : 'none')};
  display: flex;
  flex-direction: row;
  width: 100%;
  &:hover {
    background-color: ${IgnitusBlue};
    color: ${White};
    svg {
      fill: ${White};
    }
  }
`;

export const SideNavIcon = styled(Icon)`
  height: 1.5rem;
  width: 1.5rem;
  fill: ${IgnitusBlue};
`;

export const styledParagraph = styled.p`
  margin: 0;
  margin-left: 0.5rem;
`;
