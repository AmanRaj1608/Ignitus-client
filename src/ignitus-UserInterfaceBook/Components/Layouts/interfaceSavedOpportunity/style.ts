import styled from '@emotion/styled';

export const Grid = styled.div`
  display: grid;
  grid-template-columns: repeat(2, 4fr);
  grid-auto-rows: auto;
  grid-row-gap: 2rem;
`;
