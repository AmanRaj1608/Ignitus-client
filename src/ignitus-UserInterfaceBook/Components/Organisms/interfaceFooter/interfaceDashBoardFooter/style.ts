import styled from '@emotion/styled';

export const DashBoardFooterContainer = styled.div`
  padding: 3rem;
`;
