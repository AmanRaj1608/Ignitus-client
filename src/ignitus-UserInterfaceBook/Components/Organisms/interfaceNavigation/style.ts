import styled from '@emotion/styled';

export const NavigationContainer = styled.div`
  display: flex;
  flex-direction: row;
  justify-content: space-evenly;
  flex: 1;
`;
