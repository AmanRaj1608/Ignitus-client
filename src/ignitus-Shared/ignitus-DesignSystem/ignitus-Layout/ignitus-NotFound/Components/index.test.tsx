import React from 'react';
import { mount, render } from 'enzyme';
import emotionSerializer from 'jest-emotion';
import { Notfound } from './index';
import * as S from '../style';

let wrapper;

expect.addSnapshotSerializer(emotionSerializer);

describe('<Notfound />', () => {
  beforeEach(() => {
    wrapper = mount(<Notfound />);
  });

  it('should render', () => {
    wrapper = render(<Notfound />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render Eyes, Message, Button, AnimationText components', () => {
    expect(wrapper.find(S.Eyes).length).toBe(2);
    expect(wrapper.find(S.Message).length).toBe(1);
    expect(wrapper.find(S.StyledButton).length).toBe(1);
    expect(wrapper.find(S.AnimationText).length).toBe(1);
  });
});
