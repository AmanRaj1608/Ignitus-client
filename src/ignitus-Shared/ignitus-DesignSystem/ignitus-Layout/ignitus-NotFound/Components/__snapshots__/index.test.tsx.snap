// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Notfound /> should render 1`] = `
@keyframes animation-0 {
  0% {
    -webkit-transform: translate(1px,1em);
    -ms-transform: translate(1px,1em);
    transform: translate(1px,1em);
  }

  50% {
    -webkit-transform: translate(0,1em);
    -ms-transform: translate(0,1em);
    transform: translate(0,1em);
  }

  100% {
    -webkit-transform: translate(-1px,1em);
    -ms-transform: translate(-1px,1em);
    transform: translate(-1px,1em);
  }
}

@keyframes animation-1 {
  0%, 30%, 55%, 90%, 100% {
    -webkit-transform: translate(0,0);
    -ms-transform: translate(0,0);
    transform: translate(0,0);
  }

  10%, 25% {
    -webkit-transform: translate(0,20px);
    -ms-transform: translate(0,20px);
    transform: translate(0,20px);
  }

  65% {
    -webkit-transform: translate(-20px,0);
    -ms-transform: translate(-20px,0);
    transform: translate(-20px,0);
  }

  80% {
    -webkit-transform: translate(20px,0);
    -ms-transform: translate(20px,0);
    transform: translate(20px,0);
  }
}

@keyframes animation-2 {
  to {
    text-indent: -373px;
  }
}

.emotion-9 {
  background: #F7F7F7;
}

.emotion-2 {
  display: block;
  position: relative;
  width: 200px;
  margin: 10% auto 0;
  -webkit-animation: animation-0 0.2s infinite;
  animation: animation-0 0.2s infinite;
  height: 100%;
}

.emotion-2::after {
  content: '';
  width: 20px;
  height: 20px;
  background: white;
  position: absolute;
  top: 30px;
  left: 25px;
  border-radius: 50%;
  box-shadow: 125px 0 0 white;
  -webkit-animation: animation-1 2.5s infinite;
  animation: animation-1 2.5s infinite;
}

.emotion-0 {
  position: relative;
  display: inline-block;
  background: #000066;
  width: 75px;
  height: 80px;
  border-radius: 50%;
  -webkit-transform: rotate(45deg);
  -ms-transform: rotate(45deg);
  transform: rotate(45deg);
}

.emotion-0::after {
  content: '';
  position: absolute;
  border-bottom: 2px solid #000066;
  width: 70px;
  height: 50px;
  left: 0px;
  bottom: -10px;
  border-radius: 50%;
}

.emotion-0::before {
  bottom: auto;
  top: -100px;
  -webkit-transform: rotate(45deg);
  -ms-transform: rotate(45deg);
  transform: rotate(45deg);
  left: 0;
}

.emotion-0:nth-of-type(2) {
  float: right;
  -webkit-transform: rotate(-45deg);
  -ms-transform: rotate(-45deg);
  transform: rotate(-45deg);
}

.emotion-0:nth-of-type(2)::after {
  left: 5px;
}

.emotion-0:nth-of-type(3) {
  display: none;
}

.emotion-3 {
  margin-top: 5rem;
  text-align: center;
}

.emotion-3::before {
  content: 'Got lost? How......?  Ahhhh...!';
  color: #000066;
  width: 120px;
  font-size: 30px;
  overflow: hidden;
  display: inline-block;
  white-space: nowrap;
  -webkit-animation: animation-2 2s infinite steps(3);
  animation: animation-2 2s infinite steps(3);
}

.emotion-8 {
  text-align: center;
  margin-bottom: 6rem;
}

.emotion-6 {
  color: #000066;
}

.emotion-4 {
  font-size: 16px;
  font-weight: 400;
  color: #2D2D2D;
  margin: 0;
  color: #000066;
  font-size: 16px;
}

.emotion-7 {
  border-radius: 5px;
  border: none;
  cursor: pointer;
  outline: none;
  padding: 16px 32px;
  text-align: center;
  line-height: 1rem;
  margin: 0.5rem;
  outline: none;
  font-size: 18px;
  font-weight: 400;
  -webkit-transition: all 0.3s ease;
  transition: all 0.3s ease;
  outline: none;
  color: #ffffff;
  background-color: #000066;
  padding: 16px 32px;
  font-size: 18px;
  border-radius: 5px;
  border: none;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-7:hover,
.emotion-7:focus {
  box-shadow: 0 2px 4px 0 rgba(0,0,0,0.05);
}

.emotion-7 a {
  color: #ffffff;
}

.emotion-7:hover,
.emotion-7:focus {
  background-color: #F9BF64;
  color: #000066;
  outline: none;
}

.emotion-7:hover a,
.emotion-7:focus a {
  color: #000066;
}

.emotion-7:hover svg,
.emotion-7:focus svg {
  fill: #000066;
}

.emotion-7 a {
  color: #ffffff;
}

.emotion-7:hover {
  color: #000066;
}

.emotion-7:hover a {
  color: #000066;
  -webkit-text-decoration: none;
  text-decoration: none;
}

<div
  class="emotion-9"
>
  <div
    class="emotion-2"
  >
    <div
      class="emotion-0"
    />
    <div
      class="emotion-0"
    />
  </div>
  <p
    class="emotion-3"
  />
  <div
    class="emotion-8"
  >
    <div
      class="emotion-6"
    >
      <p
        class="emotion-4"
      >
        Looks like you're lost!
      </p>
      <p
        class="emotion-4"
      >
        The page you are looking for is not available! 
        <span
          aria-label="hammer"
          role="img"
        >
          ðŸ”¨
        </span>
      </p>
    </div>
    <button
      class="emotion-7"
    >
      <a
        href="/"
      >
         Home Page 
      </a>
    </button>
  </div>
</div>
`;
